strict-lens/README.md

```diff
@@ -0,0 +1,102 @@
+![build status](https://travis-ci.org/meiersi/strict-base-types.svg?branch=master)
```

Comment:
Most of this file is obsolete now, I think it's OK to drop it.

Reply:
Goot catch.